services/movie_session.py

```diff
@@ -1,10 +1,14 @@
 from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
 
-from db.models import MovieSession
+import init_django_orm  # noqa: F401
+from db.models import MovieSession, Ticket
 
 
 def create_movie_session(
-    movie_show_time: str, movie_id: int, cinema_hall_id: int
+        movie_show_time: str,
```

Comment:
Consider using `datetime` objects for `movie_show_time` instead of strings to ensure proper handling of date and time formats.

Reply:
This has already been written by the mentor